modules/minimum-required-permissions-ipi-gcp-xpn.adoc

```diff
@@ -25,6 +25,18 @@ Ensure that the host project applies one of the following configurations to the
 * `roles/compute.securityAdmin`
 ====
 
+.Required permissions for creating service account resources
```

Comment:
In fact, these permissions are for getting/setting the project's IAM policy in the host project, e.g. granting the service account(s) the required permissions in the host project.